moped-toolbox/council_districts_load/insert_council_districts.py

```diff
@@ -0,0 +1,124 @@
+"""Downloads council districts geojson from AGOL and inserts it into Moped"""
+import argparse
+import requests
+from secrets import HASURA
+
+
+DISTRICTS_URL = "https://services.arcgis.com/0L95CJ0VTaxqcmED/ArcGIS/rest/services/BOUNDARIES_single_member_districts/FeatureServer/0/query?where=1%3D1&objectIds=&time=&geometry=&geometryType=esriGeometryEnvelope&inSR=&spatialRel=esriSpatialRelIntersects&resultType=none&distance=0.0&units=esriSRUnit_Meter&relationParam=&returnGeodetic=false&outFields=*&returnGeometry=true&returnCentroid=false&featureEncoding=esriDefault&multipatchOption=xyFootprint&maxAllowableOffset=&geometryPrecision=8&outSR=4326&defaultSR=&datumTransformation=&applyVCSProjection=false&returnIdsOnly=false&returnUniqueIdsOnly=false&returnCountOnly=false&returnExtentOnly=false&returnQueryGeometry=false&returnDistinctValues=false&cacheHint=false&orderByFields=&groupByFieldsForStatistics=&outStatistics=&having=&resultOffset=&resultRecordCount=&returnZ=false&returnM=false&returnExceededLimitFeatures=true&quantizationParameters=&sqlFormat=none&f=pgeojson&token="
```

Comment:
Esri adds in all these default params when you build a query from their UI. the params of note:

- geometryPrecision=8: cap coordinates at 8 decimal places
- outSR=4326: use WGS84 coordinates
- f=pgeojson: return features as geojson